### PR TITLE
+ Turkish Language Added.

### PR DESCRIPTION
**Pull request by ramazansancar**
+ Bug fix
+ Empty Line Cleared.

## [Suggestion] How about using Weblate for translation?
- [Web Site](https://weblate.org/)
- [Open Source Repo](https://github.com/WeblateOrg/weblate)
- [Demo (User-friendly interface)](https://hosted.weblate.org/)

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -88,4 +88,4 @@ const DEFAULT_CONFIG = {
 // needs to contain actual changes, not all default values - and these
 // stay up-to-date automatically.
 // There might not be a user-defined config, default to an empty object.
-export default deepmerge(DEFAULT_CONFIG, (window.owntracks || {}).config || {});
+export default deepmerge(DEFAULT_CONFIG, (window.owntracks || {}).config || {});
```
Comment (linusg) [inline src/config.js:91]: This removes a newline at the end - please don't.

File: src/locales/tr-TR.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "Cancel": "İptal",
+  "Image of {deviceName}": "{deviceName} resmi",
+  "Timestamp": "Zaman Damgası",
+  "Location": "Konum",
+  "Address": "Adres",
+  "Battery": "Batarya",
+  "Speed": "Hız",
+  "Regions:": "Bölgeler:",
+  "WiFi": "WiFi"
+}
```
Comment (linusg) [inline src/locales/tr-TR.json:38]: Missing newline at end of file

**Review by linusg — CHANGES_REQUESTED** [review]

File: src/config.js (hunk 1 above)
Comment (ramazansancar) [reply src/config.js]: @linusg I haven't made any changes here but it looks like it has been done :/

**Review by ramazansancar — COMMENTED** [review]
resolved

**Review by linusg — APPROVED** [review]
Thanks!

**Pull request merged**